explorer/types/explorertypes.go

```diff
@@ -173,6 +173,7 @@ type TrimmedTxInfo struct {
 // TxInfo models data needed for display on the tx page
 type TxInfo struct {
 	*TxBasic
+	TxVersion        int32
```

Comment:
This works.  I'm cooking up a PR to do some mempool and DB work, and in that I'm moving `TxVersion` to `TxBasic` and renaming it to just `Version`.  Just a heads up.  This is good like this for now.